lighthouse-cli/test/cli/run-test.js

```diff
@@ -145,4 +145,25 @@ describe('Parsing --chrome-flags', () => {
       ['--spaces=1 2 3 4', '--debug=false', '--verbose', '--more-spaces=9 9 9']
     );
   });
+
+  it('handles muliple --chrome-flags', () => {
+    assert.deepStrictEqual(
+      parseChromeFlags(['--no-sandbox', '--log-level=0']),
+      ['--no-sandbox', '--log-level=0']
+    );
+  });
+
+  it('removes wrapping single quotes', () => {
+    assert.deepStrictEqual(
+      parseChromeFlags('\'--no-sandbox --log-level=0\''),
+      ['--no-sandbox', '--log-level=0']
+    );
+  });
+
+  it('removes wrapping double quotes', () => {
+    assert.deepStrictEqual(
+      parseChromeFlags('"--no-sandbox --log-level=0"'),
+      ['--no-sandbox', '--log-level=0']
+    );
+  });
```

Comment:
```suggestion
  });
  
  it('handles muliple --chrome-flags with unnecessary quotation', () => {
    assert.deepStrictEqual(
      parseChromeFlags(['"--no-sandbox --headless"', '--log-level=0']),
      ['--no-sandbox', '--headless', '--log-level=0']
    );
  });
```

I *think* this one is going to fail currently

Reply:
also might want to throw in a test just to document how it handles partially malformed input now that we're doing a little more mutation to the input?

I'm thinking of something like 

`"--headless --host-resolver-rules="MAP www.example.org:443 127.0.0.1:8443"` where they forget to end quote

Reply:
I added this feature in the last commit - could you please check if understood it correctly?

Reply:
Oh sorry I wasn't trying to ask you to try to handle the unterminated quote case! just to document what happens in failure modes